lib/paper_trail/version_concern.rb

```diff
@@ -5,7 +5,9 @@ module VersionConcern
     extend ::ActiveSupport::Concern
 
     included do
-      belongs_to :item, :polymorphic => true
+      opts = {:polymorphic => true}
+      opts[:required] = false if ActiveRecord::Base.respond_to? :belongs_to_required_by_default
```

Comment:
Do we need both this change (required: false) *and* the change to the default `recording_order`?